### ISSUE TITLE
clarify presentTitleProperty

### ISSUE THREAD

**Issue by tomkralidis** [opengeospatial/ets-cat30#16]
Given https://github.com/opengeospatial/ets-cat30/commit/c4348abd92035f0a849286d4e1eb6f7ef092092d, I'm now getting a failure when testing against [pycsw](http://demo.pycsw.org/cite/csw):

```
<![CDATA[XPath expression cannot be evaluated: not(exists(csw:SummaryRecord[dc:type or dc:subject or dc:format or ows:BoundingBox])). {1}]]>
```
Request: http://demo.pycsw.org/cite/csw?elementName=tns:title&request=GetRecords&service=CSW&typeNames=Record&namespace=xmlns%28tns%3Dhttp://purl.org/dc/elements/1.1/%29&version=3.0.0

What is the rationale of this assertion?

**Comment by pandzel** [comment]
I am getting the same. Moreover, the root cause of this error is an exception being thrown with the following message:

**javax.xml.transform.TransformerException: Could not find function: exists**

Perhaps, 'exists' function is XPath 2.0, and many frameworks like Java naively supports only 1.0 version which doesn't have 'exists' function.

I would suggest to replace XPath expression with something which doesn't use 'exists', like:

**count(csw:SummaryRecord[dc:type or dc:subject or dc:format or ows:BoundingBox])=0**

**Comment by tomkralidis** [comment]
Looks like this has been addressed in 7dffe4110213e95c5feaa16efde7dae5d98e620a.  All (pycsw) tests pass again -- thanks!
